pkg/controller/spinnakervalidating/spinnakervalidating_controller.go

```diff
@@ -92,10 +128,15 @@ func (v *spinnakerValidatingController) Handle(ctx context.Context, req types.Re
 		Ctx:    ctx,
 		Client: v.client,
 		Req:    req,
+		Log:    log,
 	}
-	if err := validate.Validate(svc, opts); err != nil {
-		log.Error(err, "SpinnakerService validation failed", "metadata.name", svc)
-		return admission.ErrorResponse(http.StatusBadRequest, err)
+	log.Info("Starting validation")
+	errors := validate.Validate(svc, opts)
+	if len(errors) > 0 {
+		for _, e := range errors {
+			log.Error(e, fmt.Sprintf("SpinnakerService validation failed: %s", e.Error()), "metadata.name", svc)
+		}
+		return admission.ErrorResponse(http.StatusBadRequest, errors[0])
```

Comment:
I'd rather have a new `error` with the messages aggregated. This is what the user will see as a result of `kubectl`, they shouldn't have to dig through the logs.